apps/alarm/data-migration.js

```diff
@@ -80,7 +79,14 @@ module.exports = function getAlarms (activity, callback) {
       activity: activity,
       intent: intent,
       businessParams: businessParams || {},
-      callback: (res) => {
+      callback: (err, res) => {
```

Comment:
Since signature of params.callback has been changed, should all the invocation be updated?

Reply:
alarm-core.js && data-migration.js may use it, alarm-core.js has no callback param,so have been updated all the invocation